armhf.Dockerfile

```diff
@@ -8,6 +8,8 @@ RUN apt-get install -y \
         git \
         libprocps-dev:armhf \
         libssl-dev:armhf \
+        libz-dev:armhf \
```

Comment:
These are needed by sync, and it seemed best to use a common docker image for both.